js/ui/popup/popup_position_controller.js

```diff
@@ -46,7 +43,11 @@ class PopupPositionController extends OverlayPositionController {
     }
 
     get outsideDragFactor() {
-        return this._outsideDragFactor;
+        if(this._props.dragOutsideBoundary) {
+            return 1;
```

Comment:
What does 1 mean?

Reply:
>What does 1 mean?
 It is some factor using for drag and drop